Content.Server/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -31,6 +34,39 @@ public override bool IsPlaceable
             }
         }
 
+        public override bool PlaceCentered
+        {
+            get => _placeCentered;
+            set
+            {
+                if (_placeCentered == value)
+                {
+                    return;
+                }
+
+                _placeCentered = value;
+
+                Dirty();
+            }
+        }
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public override Vector2 PositionOffset
+        {
+            get => _positionOffset;
+            set
+            {
+                if (_positionOffset == value)
```

Comment:
This is never gonna be true unless it's the same ref so IMO check for EqualsApprox.

Reply:
I've replaced the condition with EqualsApprox as suggested, thank you